pepper-apis/src/main/java/org/broadinstitute/ddp/Housekeeping.java

```diff
@@ -253,11 +255,26 @@ public static void main(String[] args) {
                     JdbiQueuedEvent queuedEventDao = apisHandle.attach(JdbiQueuedEvent.class);
                     // first query the full list of pending events, shuffled to avoid event starvation
                     LOG.info("Querying pending events");
-                    List<QueuedEventDto> pendingEvents = eventDao.getQueuedEvents();
+                    List<QueuedEventDto> pendingEvents = eventDao.findPublishableQueuedEvents();
                     LOG.info("Found {} events that may be publishable", pendingEvents.size());
                     Collections.shuffle(pendingEvents);
 
                     for (QueuedEventDto pendingEvent : pendingEvents) {
+                        if (pendingEvent.getParticipantGuid() != null) {
+                            User participant = apisHandle.attach(UserDao.class)
+                                    .findUserByGuid(pendingEvent.getParticipantGuid())
+                                    .orElse(null);
+                            if (participant == null) {
```

Comment:
This should help with https://broadinstitute.atlassian.net/browse/DDP-4043

Reply:
Ideally I think we should never run into user not found in queued events. Deleting user should warrant dependencies like queued events deletion too. right? However if you ever run into this I think may be log as error LOG.error, rather than warning because we can investigate.

Reply:
You're right -- I think we'll encounter this situation if we have already published the message to pubsub. It looks like we already have code in the receiver to handle that case. I'll a `log.error()` here.

Reply:
done